lib/SimpleSAML/Configuration.php

```diff
@@ -198,7 +198,7 @@ public static function setConfigDir($path, $configSet = 'simplesaml')
 
 
     /**
-     * Load a configuration file from a configuration set.
+     * Load a configuration file from a configuration set.XM
```

Comment:
oops. Removed it. Probably my subconscious doing a keyboard shortcut that didn't exist in my editing app :)